genai-perf/genai_perf/profile_data_parser/image_retrieval_profile_data_parser.py

```diff
@@ -76,9 +81,17 @@ def _parse_requests(self, requests: dict) -> ImageRetrievalMetrics:
         benchmark_duration = (max_res_timestamp - min_req_timestamp) / 1e9  # to seconds
         request_throughputs = [len(requests) / benchmark_duration]
 
-        return ImageRetrievalMetrics(
+        self._image_metric = ImageRetrievalMetrics(
```

Comment:
is there a reason why this needs to be an instance attribute?

Reply:
No, nice catch. 
Updated in the new version.